cmd/logs.go

```diff
@@ -51,7 +57,7 @@ k8s-multi-pod logs "app=hello" -c ingress`,
 
 		labelSelector := args[0]
 
-		fmt.Println("Retrieving logs...this could take a minute.")
+		fmt.Printf("\nRetrieving logs...this could take a minute.\n\n")
```

Comment:
Not sure why you're using `fmt.Printf` because you're not using a format string.